src/windows/title_editor.py

```diff
@@ -432,10 +432,14 @@ def btnFont_clicked(self):
         if ok and font is not oldfont:
             self.qfont = font
             fontinfo = QtGui.QFontInfo(font)
+            oldfontinfo = QtGui.QFontInfo(oldfont)
             self.font_family = fontinfo.family()
             self.font_style = fontinfo.styleName()
             self.font_weight = fontinfo.weight()
-            self.font_size_pixel = fontinfo.pixelSize()
+            if (oldfontinfo.pixelSize() != 0):
+                self.font_size_ratio = fontinfo.pixelSize() / oldfontinfo.pixelSize()
+            else:
+                self.font_size_ratio = 0
```

Comment:
If we set `font_size_ratio` = 0, won't this eventually lead to a 0 sized `new_font_size_pixel`? I'm unclear why the oldfontinfo.pixelSize would be zero to begin with. If we don't detect a valid oldfontsize, should the ratio default to 1, and just stay the same size? What do you think?

Reply:
I'm really only afraid of a user launching the editor and accidentally making the font-size 0 (and disappearing), lol

Reply:
My initial thinking was that svg files can be modified by users, or user supplied, and might contain invalid values like 0, which will result in an error when we use it as a denominator. But it might not be an issue since we are handling pre-defined templates.

Sure, let's default the font size to 1. Since it is already initialized to 1, we will perform the division when `oldfontinfo.pixelSize() > 0`